pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -154,8 +154,13 @@ func configureFromProviderConfig(pushCtx *model.PushContext, proxy *model.Proxy,
 
 	switch provider := providerCfg.Provider.(type) {
 	case *meshconfig.MeshConfig_ExtensionProvider_Zipkin:
-		tracing, err = buildHCMTracing(pushCtx, envoyZipkin, provider.Zipkin.GetService(),
-			provider.Zipkin.GetPort(), provider.Zipkin.GetMaxTagLength(), zipkinConfigGen, serviceCluster)
+		tracing, err = buildHCMTracingConfig(envoyZipkin, provider.Zipkin.GetMaxTagLength(), func() (*anypb.Any, error) {
```

Comment:
Looks like some are still using the old `buildHCMTracing` with pushCtx and not you added `buildHCMTracingConfig` - Are you planning to consolidate them? It is confusing to have both with slightly different name?

Reply:
yes, plan to do it later.